recipe/meta.yaml

```diff
@@ -1,35 +1,39 @@
-{% set version = "1.16.2" %}
+{% set name = "automake" %}
+{% set version = "1.16.5" %}
+{% set checksum = "07bd24ad08a64bc17250ce09ec56e921d6343903943e99ccf63bbf0705e34605" %}
+{% set build = "0" %}
 
 package:
-  name: automake
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  url: http://ftp.gnu.org/gnu/automake/automake-{{ version }}.tar.gz
-  sha256: b2f361094b410b4acbf4efba7337bdb786335ca09eb2518635a09fb7319ca5c1
+  url: https://ftp.gnu.org/gnu/automake/automake-{{ version }}.tar.gz
+  sha256: {{ checksum }}
 
 build:
-  number: 0
+  number: {{ build }}
   skip: true  # [win]
 
 requirements:
   host:
     - m4
     - autoconf
-    - perl
+    - perl 5.*
     - make
   run:
     - m4
     - autoconf
-    - perl
+    - perl 5.*
 
 test:
   commands:
     - automake --help
 
 about:
-  home: http://www.gnu.org/software/automake/
-  license: GPL 3
+  home: https://www.gnu.org/software/automake/
+  license: GPL-2.0-only
```

Comment:
Note that the README file states: 
"This program is free software; you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation; either version 2, or (at your option)
any later version."

So the source code is a bit ambiguous.  Not sure what the right answer here BUT the COPYING file is pretty clear.